custom-event-polyfill.js

```diff
@@ -42,3 +42,5 @@ try {
   CustomEvent.prototype = window.Event.prototype;
   window.CustomEvent = CustomEvent; // expose definition to window
 }
+
+return window.CustomEvent;
```

Comment:
Please remove this line. This is a polyfill, not a ponyfill.  This module exports a function if you need that: https://www.npmjs.com/package/custom-event